examples/ios/swift/Migration/AppDelegate.swift

```diff
@@ -17,39 +17,6 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import UIKit
-import RealmSwift
-
-// Old data models
-/* V0
-class Person: Object {
-    @objc dynamic var firstName = ""
-    @objc dynamic var lastName = ""
-    @objc dynamic var age = 0
-}
-*/
-
-/* V1
-class Person: Object {
-    @objc dynamic var fullName = ""        // combine firstName and lastName into single field
-    @objc dynamic var age = 0
-}
-*/
-
-/* V2 */
-class Pet: Object {
-    @objc dynamic var name = ""
-    @objc dynamic var type = ""
-}
-
-class Person: Object {
-    @objc dynamic var fullName = ""
-    @objc dynamic var age = 0
-    let pets = List<Pet>() // Add pets field
-}
-
-func bundleURL(_ name: String) -> URL? {
-    return Bundle.main.url(forResource: name, withExtension: "realm")
-}
 
 #if !swift(>=4.2)
```

Comment:
I don't know. Do we support Swift < 4.2 still?

Reply:
@jsflax How to decide this?

Reply:
We support the minimum version of Swift that our minimum supported Xcode version supports (and we support 1 major version beneath the current Xcode version): https://developer.apple.com/support/xcode/.  So, yes.

Reply:
However, I think the better question is, are we still using `UIApplicationLaunchOptionsKey`?

Reply:
`UIApplicationLaunchOptionsKey` was substituted by `UIApplication.LaunchOptionsKey` in Swift 4.2. So `UIApplication.LaunchOptionsKey` will probably not work is someone is using Swift 4.0 on Xcode 11.0, which is still supported according to your commend above.
I don't understand how the `typealias` solves this though since `UIApplicationLaunchOptionsKey` is not nested inside `UIApplication` anyway but a global key.

Reply:
But does the example still need it given your changes?

Reply:
@jsflax  It doesn't have anything to do with the examples. It's defined in all `AppDelegate` files in all examples since it's part of

```
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
```

Reply:
So the answer was already given by you by saying we still support Swift <4.2. :)
Until Xcode 13 is has to stay there then.

Reply:
@DominicFrei has tested this locally and it no longer seems to be used and can be removed.